app/admin/user.rb

```diff
@@ -1,4 +1,6 @@
 ActiveAdmin.register User do
+  actions :all, except: [:destroy]
```

Comment:
about about explicitly setting which actions we want rather than which one we don't want? that is the style I usually use for `before_action`s or `config/routes` -- the result is the same but it's nice to see which actions are enabled explicitly!